src/libraries/restore/runtime/runtime.depproj

```diff
@@ -90,6 +89,7 @@
       <BinPlaceItem Include="@(MonoIncludeFiles)">
         <DestinationSubDirectory>include/%(RecursiveDir)</DestinationSubDirectory>
       </BinPlaceItem>
+      <BinPlaceItem Remove="@(BinPlaceItem)" Condition="'$(RuntimeFlavor)' != 'Mono' and '%(BinPlaceItem.Filename)' == 'System.Private.CoreLib'" />
```

Comment:
curious, why are you removing coreclr's S.P.Corelib?

Reply:
Because it was being added in both managed and native folders. I do believe that there was an underlying issue here when constructing our packs for non-mono, but that can be addressed in a separate PR.